app/services/order_fees_handler.rb

```diff
@@ -19,6 +19,7 @@ def recreate_all_fees!
 
       create_line_item_fees!
       create_order_fees!
+      tax_enterprise_fees!
```

Comment:
I'm a bit concerned about the potential lock contention caused by this growing transaction. Is there a chance we can move the calls to `order.updater` below outside the block? It looks like a sensible trade-off to me.

Reply:
Yeah, I'll take a look at it. I'm not 100% convinced the lock is even useful here...?